services/server/test/integration/database.spec.ts

```diff
@@ -197,19 +203,31 @@ describe("Verifier Alliance database", function () {
     chai.expect(parseInt(row.block_number)).to.equal(blockNumber);
     chai.expect(parseInt(row.transaction_index)).to.equal(txIndex);
 
-    ///
+    // Check Keccak256 for code.code_hash_keccak
+    chai
+      .expect(`0x${toHexString(row.compiled_creation_code_hash_keccak)}`)
+      .to.equal(keccak256(bytesFromString(testCase.compiled_creation_code)));
```

Comment:
Actually... I think we should not be calculating values when asserting test cases. Imagine for example there's a bug in the keccak function. Or it could be that the output value is different when you input a `Buffer` vs `string` to the `keccak` function. 

Since we are also calculating the `code_hash_keccak` field in the same way as this line in our server, this assertion is actually self-affirming. I think the test cases should contain the actual hash values and not be calculated on the fly. 

This doesn't have to be done in this PR but we can branch out this to another issue.

Is it clear what I mean?

Reply:
It is clear, from my pov we can postpone this to another PR and more analysis